pkg/handler/scale_jobs.go

```diff
@@ -0,0 +1,87 @@
+package handler
+
+import (
+	"fmt"
+
+	log "github.com/Sirupsen/logrus"
+	keda_v1alpha1 "github.com/kedacore/keda/pkg/apis/keda/v1alpha1"
+	batchv1 "k8s.io/api/batch/v1"
+	apiv1 "k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (h *ScaleHandler) scaleJobs(scaledObject *keda_v1alpha1.ScaledObject, isActive bool, scaleTo int64, maxScale int64) {
+	// TODO: get current job count
+	log.Println("Scaling Jobs")
+
+	if isActive {
+		log.Println("Scaler is active")
+		now := meta_v1.Now()
+		scaledObject.Status.LastActiveTime = &now
+		h.updateScaledObject(scaledObject)
+		h.createJobs(scaledObject, scaleTo, maxScale)
+	} else {
+		log.Debugf("scaledObject (%s) no change", scaledObject.GetName())
+	}
+	return
+}
+
+func (h *ScaleHandler) createJobs(scaledObject *keda_v1alpha1.ScaledObject, scaleTo int64, maxScale int64) {
+	var containers []apiv1.Container
+	for _, c := range scaledObject.Spec.ConsumerSpec.Containers {
+		containers = append(containers, apiv1.Container{
+			Name:  c.Name,
+			Image: c.Image,
+			Env:   c.Env,
+		})
+	}
+
+	if scaleTo > maxScale {
+		scaleTo = maxScale
+	}
+	log.Printf("Creating %d jobs", scaleTo)
+
+	for i := 0; i < int(scaleTo); i++ {
+
+		job := &batchv1.Job{
+			ObjectMeta: meta_v1.ObjectMeta{
+				GenerateName: scaledObject.GetName() + "-",
```

Comment:
@stgricci does this need to be unique? Can multiple instances get triggered together?

Reply:
@patnaikshekhar I believe it does have to be unique / single job creation, there is no CreateCollection type method in client-go[0] for jobs. Parallelism does exist for jobs, but this could potentially impact normal operation of more complex jobs (jobs that require partial completion in order for them to be considered successful)

[0] https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/batch/v1/job.go